symbols/overlay29.yml

```diff
@@ -1085,6 +1085,40 @@ overlay29:
         r2: y position
         r3: flags
         stack[0]: visibility flag
+    - name: PrepareTrapperTrap
+      address:
+        NA: 0x22EDBD4
+      description: |-
+        Saves the relevant information in the dungeon struct to later place a trap at the
+        location of the entity. (Only called with trap ID 0x19, but could be used with others).
```

Comment:
> Only called with trap ID 0x19

Even when a trapper orb is used?

Reply:
This function is only called by DoMoveTrapperOrb and it only uses 0x19. The function that creates traps seems to randomly select a trap ID that's not the wonder tile trap if it's above 0x18.

Reply:
Oh, so it uses 0x19 as a way to make the function pick a random trap. I see.

Reply:
Trap ID 0x19 is `TRAP_NONE`. Can we use that terminology here instead? And update the comment over `TRAP_NONE` with this additional use?